src/test/static/analysis/behaviors/analysis.json

```diff
@@ -39,7 +39,7 @@
             }
           },
           "metadata": {},
-          "type": "number",
+          "type": "?number|undefined",
```

Comment:
I'm in favor of everything about this PR except emitting types like this in `analysis.json`. It hurts readability, but I also think this kind of precision is inappropriate given the level of checking we have of types in elements code. This level of precision says that if you don't see `?` or `undefined` then you can expect the value to be defined and not null.

But I don't think that we can generally say that with confidence. But maybe I'm wrong, and our types are more reliable I think?

Reply:
Discussed offline. Updated to always emit as `<type>|null|undefined`, since that's still a valid Closure type, but also more intuitive to people not familiar with Closure type syntax.